src/auth0_server_python/auth_server/server_client.py

```diff
@@ -290,7 +299,7 @@ async def complete_interactive_login(
 
         # Build a token set using the token response data
         token_set = TokenSet(
-            audience=token_response.get("audience", "default"),
+            audience=transaction_data.audience or "default",
```

Comment:
Is this a bug fix? (paired with the changes [above](https://github.com/auth0/auth0-server-python/pull/57/files#diff-f89e94b52d5636391323612572af43429518b8d6aa24a4f9e60e9c8489d3bc7bR167))

Reply:
Actually, this seems to not realy be a bug fix as we did not have the concept of more audiences.